### ISSUE TITLE
[Feature] Is there any way to perform transfer learning in MMCV

### ISSUE THREAD

**Issue by yasar-rehman** [open-mmlab/mmclassification#378]
so far I have seen that in mmcv if you fine-tune a model with pre-trained weights it basically fine tune the whole model. I would like to know if there is any way to freeze the backbone layers and only train the final classification layer. 
Thanks

**Comment by Ezra-Yu** [comment]
You can set the [frozen_stages](https://github.com/open-mmlab/mmclassification/blob/33a4c0fe9f6642536bd969d9d140557b827705a9/mmcls/models/backbones/resnet.py#L409) parameter in backbone config. example can be seen in this [config](https://github.com/open-mmlab/mmdetection/blob/ba286696eb1fee2e02f3aae04164f4e79296ea80/configs/_base_/models/faster_rcnn_r50_fpn.py#L9).

frozen_stages (int):  Stages_to be frozen(stop grad and set eval mode). -1 means not freezing any parameters. Default: -1.

**Comment by yasar-rehman** [comment]
Thanks @Ezra-Yu  for the prompt response! Can it accept the "bool" (True/False) keyword. Let's say for example if I want to freeze all the layers in the backbone --instead of specifying the number of frozen_stages as int-- one could just pass fronzen_stages as True/False to freeze all the layers?

**Comment by Ezra-Yu** [comment]
No! fronzen_stages can only accept int value. “fronzen_stages = n” means fronze the N levels in front instead of the specifying nth level.

for example, if  fronzen_stages = 0, the 0th leval (means stem level) is fronzen.
 if  fronzen_stages = 1, the 0th leval (means stem level)  and the stage 1 are fronzen.

The resnet(etc.) backbone have 5 leaves, including stem, stage1, stage 2,  stage3, stage 4.

**Comment by yasar-rehman** [comment]
Gotcha! Thanks for the quick help
